.travis.yml

```diff
@@ -58,16 +58,16 @@ deploy:
       repo: aeternity/epoch
       branch: env/dev2
       condition: "${JOB:?} = package && ${TRAVIS_OS_NAME:?} = linux"
-# matrix:
-#   include:
-#     - os: osx
-#       osx_image: xcode8.3 # [`xcode8.3` is Xcode 8.3.3 on OS X 10.12](https://docs.travis-ci.com/user/reference/osx#OS-X-Version)
-#       language: generic
-#       env: JOB=test
-#     - os: osx
-#       osx_image: xcode8.3 # [`xcode8.3` is Xcode 8.3.3 on OS X 10.12](https://docs.travis-ci.com/user/reference/osx#OS-X-Version)
-#       language: generic
-#       env: JOB=package
+matrix:
+  include:
+    - os: osx
+      osx_image: xcode8.3 # [`xcode8.3` is Xcode 8.3.3 on OS X 10.12](https://docs.travis-ci.com/user/reference/osx#OS-X-Version)
+      language: generic
+      env: JOB=test
```

Comment:
Please enable only package job unless you are confident OSX workers consume jobs enough.

Reply:
@lucafavatella Re-enabling OSX builds is mainly for statistics purposes ATM.
See the reference email from @travis in #devops.

However, it looks like it won't start for 20 minutes already, which is still a big problem.

Reply:
yea that is why I suggest marking it in allow_failures with fast_finish. I assume this will still be ok for statistics.